test/TestLogger.UnitTests/TestDoubles/JsonTestResultSerializer.cs

```diff
@@ -86,8 +88,8 @@ private Test CreateTest(TestResultInfo result)
         {
             return new ()
             {
-                FullyQualifiedName = result.TestCase.FullyQualifiedName,
-                DisplayName = result.TestCase.DisplayName,
+                ////FullyQualifiedName = result.TestCase.FullyQualifiedName,
```

Comment:
Is this intentional?

Reply:
I didn't intend to leave it commented. 

None of the loggers that depend on this one, depend on `FullyQualifiedName` so I removed it from `TestResultInfo`